src/test/resources/journeys.conf

```diff
@@ -24,6 +24,14 @@ journeys {
     ]
   }
 
+  v1-large-pdf-journey = {
+    description = "V1 Upload large pdf"
+    load = 12  // 97% of journeys per second
```

Comment:
I guess the load is JPS. How is this 97% calculated reversely?

Reply:
I think it's not actually 97% now, I think it's 25% of 97%. It seems to imply that 100% load is 12.36 but I don't understand any of these numbers.